src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -262,27 +383,23 @@ __wt_rts_btree_walk_btree_apply(
 
     if (modified || max_durable_ts > rollback_timestamp || prepared_updates ||
       has_txn_updates_gt_than_ckpt_snap) {
-        /* Open a handle for processing. */
-        ret = __wt_session_get_dhandle(session, uri, NULL, NULL, 0);
-        if (ret != 0)
-            WT_ERR_MSG(session, ret, "%s: unable to open handle%s", uri,
-              ret == EBUSY ? ", error indicates handle is unavailable due to concurrent use" : "");
-        dhandle_allocated = true;
-        btree = S2BT(session);
-
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-          WT_RTS_VERB_TAG_TREE "rolling back tree. uri=%s, btree_id=%" PRIu32
-                               ", modified=%s, durable_timestamp=%s > stable_timestamp=%s: %s, "
-                               "has_prepared_updates=%s, txnid=%" PRIu64
-                               " > recovery_checkpoint_snap_min=%" PRIu64 ": %s",
-          uri, btree->id, btree->modified ? "true" : "false",
-          __wt_timestamp_to_string(max_durable_ts, ts_string[0]),
+          WT_RTS_VERB_TAG_TREE
+          "rolling back tree. uri=%s"
+          ", modified=%s, durable_timestamp=%s > stable_timestamp=%s: %s, "
+          "has_prepared_updates=%s, txnid=%" PRIu64 " > recovery_checkpoint_snap_min=%" PRIu64
+          ": %s",
+          uri, modified ? "true" : "false", __wt_timestamp_to_string(max_durable_ts, ts_string[0]),
           __wt_timestamp_to_string(rollback_timestamp, ts_string[1]),
           max_durable_ts > rollback_timestamp ? "true" : "false",
           prepared_updates ? "true" : "false", rollback_txnid, S2C(session)->recovery_ckpt_snap_min,
           has_txn_updates_gt_than_ckpt_snap ? "true" : "false");
 
-        WT_ERR(__wt_rts_btree_walk_btree(session, rollback_timestamp));
+        if (S2C(session)->rts->threads == 0)
```

Comment:
If we only have 1 rts threads, is it still useful? Should we restrict the thread number to be larger than 1?